15-Appendix.Rmd

```diff
@@ -0,0 +1,144 @@
+# (APPENDIX) Appendix {-}
+
+# Conda Package Management
+
+## Unified package management for both R and Python
+
+### Faster builds
+
+CRAN[^what_is_cran] mirrors do not provide linux[^linux_note] compiled binaries
+for packages. This means a long wait when doing `install.packages` both in an R
+Studio session and when running a Docker build for a `shiny` application.
+
+### Cross Language
+
+Both `dbtools` and `s3tools` rely on Python packages through the `reticulate` R
+to Python bridge. This means that `packrat` is not enough to reproducibly and
+reliably manage all of your application's dependencies.
+
+`conda` supports managing both Python and R dependencies in a single
+`environment`. It can make sure all of these libraries are compatible with each
+other.
+
+## Installing Packages
+
+If you need to find a package name you can use the [anaconda search] to find the
+name. Run `conda install PACKAGENAME` in the `Terminal` tab to install it.
+
+### R
+
+Most CRAN (around 95%) are available through `conda`, they retain the same name
+as the CRAN package name with an addition `r-` prefix. This is to avoid clashes
+with Python packages with the same name.
+
+#### Examples:
+
+##### Install a package
+
+| `install.packages` (in R-Console) | `conda install` (in Terminal) |
+| --------------------------------- | ----------------------------- |
+| `install.packages('Rcpp')`        | `conda install r-Rcpp`        |
```

Comment:
I'm not sure if tables are the best way of representing this. Might it be better to just have this in a list?

Reply:
I quite liked the idea of the table with side-by-side previous command/new command.

Reply:
To clarify my understanding, do both commands do the same thing or is there a difference?

Reply:
@calumabarnett they both install the package `Rcpp`